test/server/util/twilio.js

```diff
@@ -22,4 +22,17 @@ describe('XML Generation', function () {
       '<?xml version="1.0" encoding="UTF-8"?><Response><Say>Call completed. Please submit the form to continue.</Say><Play loop="0">http://com.twilio.music.classical.s3.amazonaws.com/ith_chopin-15-2.mp3</Play></Response>'
     );
   });
+  it('should return TwiML that matches the the call connect template: ', () => {
+    const contactNumber = '15555555555';
+    const contactName = 'Harry Potter';
+    const connectionString = process.env.DEV_PATH;
+    const params = {
+      callId: 1,
+      campaignId: 1,
+      userId: 1
+    };
+    const { userId, campaignId, callId } = params;
```

Comment:
Why do you construct an object to turn around and deconstruct it on the next line?

Reply:
bc the function takes in an object but I need those params for the confirmation string.
I could either declare and assign constants, then put them in an object or do this.

Reply:
Let's pretend I never asked that